conf/modules.config

```diff
@@ -447,6 +497,22 @@ process {
     }
 
     withName: HAMRONIZATION_SUMMARIZE {
+        publishDir = [
+            path: { "${params.outdir}/reports/hamronization_summarize" },
+            mode: params.publish_dir_mode,
+            saveAs: { (params.run_taxonomic_classification == false) ? it : null }
```

Comment:
what has tax clsass got to do with hamronization?

Reply:
This is for the output standradization file structure. So if `run_taxonomic_classification` is turned on i dont want to outout the intermediate complete summary file (with no taxa annotations) as well as the complete summary file (with the taxa). So it only publishes when the taxa is turned off. The same was done for ampcombi and combgc